x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.test.ts

```diff
@@ -33,6 +33,12 @@ describe('KibanaLogic', () => {
       expect(KibanaLogic.values.config).toEqual({});
     });
 
+    it('gracefully handles disabled security', () => {
+      mountKibanaLogic({ ...mockKibanaValues, security: undefined } as any);
```

Comment:
(comment attached to random code)
Accessing this route http://localhost:5601/app/enterprise_search/workplace_search/sources/add/box
causes console warnings:
![image](https://user-images.githubusercontent.com/11838280/122597148-5ad08a00-d041-11eb-9753-fd903b8b2e4f.png)

Reply:
ah nice, I think it needs a `|| '...'` fallback probably?

Reply:
Addressed in https://github.com/elastic/kibana/pull/102592/commits/d77ee41383afae079f739618c4523a1a28672f5e (I found another place that would have had a similar issue and made the same change there)

Reply:
Verified, works perfectly in both places now!